commands.js

```diff
@@ -745,6 +745,12 @@ exports.commands = {
 				return this.errorReply("/" + cmd + " - Access denied for promoting/demoting to " + Config.groups[nextGroup].name + ".");
 			}
 		}
+		if ((!room.isPrivate || !room.battle || !room.isPersonal) && targetUser.locked) {
+			Monitor.log("[CrisisMonitor] " + user.name + " was automatically demoted for trying to promote locked user: " + targetUser.name + ".");
```

Comment:
It might be semi-helpful to specify what room this is referring to, considering there's quite a hand full that this could be talking about.